### PR TITLE
Tutorial for c

### PR DESCRIPTION
**Pull request by philbucher**

### CODE REVIEW

File: co_sim_io/c/co_sim_io_c.cpp [hunk 1]
```diff
@@ -98,6 +103,14 @@ CoSimIO_Info CoSimIO_ExportMesh(
     return ConvertInfo(CoSimIO::ExportMesh(ConvertInfo(I_Info), *p_container_coords, *p_container_conn, *p_container_types));
 }
 
+void CoSimIO_PrintInfo(FILE *Stream,
+    const CoSimIO_Info I_Info)
+{
+    std::stringstream buffer;
+    buffer << *static_cast<CoSimIO::Info*>(I_Info.PtrCppInfo) << std::endl;
+    fprintf(Stream, "%s", buffer.str().c_str());
```
Comment (philbucher) [inline co_sim_io/c/co_sim_io_c.cpp:111]: this leaks with valgrind @pooyan-dadvand

File: tests/co_sim_io/c/test_info_print.c [hunk 2]
```diff
@@ -12,22 +12,25 @@
...
+    CoSimIO_Info_SetInt(info, "echo_level", 6);
 
-    // CoSimIO_Info_SetInt(info, "echo_level", 6);
+    char buffer[BUFSIZ];
+    setbuf(stdout, buffer);
 
-    // COSIMIO_CHECK_TRUE(CoSimIO_Info_Has(info, "echo_level"));
+    CoSimIO_PrintInfo(stdout, info);
+ 
+    int comparison = strcmp(buffer, "CoSimIO-Info; containing 1 entries\n  name: echo_level | value: 6 | type: int\n\n");
```
Comment (philbucher) [inline tests/co_sim_io/c/test_info_print.c:30]: for some reason this also leaks

File: tests/co_sim_io/c/test_info_print.c [hunk 3]
```diff
@@ -12,22 +12,25 @@
...
 {
-    // CoSimIO_Info info = CoSimIO_CreateInfo();
+    CoSimIO_Info info = CoSimIO_CreateInfo();
 
-    // COSIMIO_CHECK_FALSE(CoSimIO_Info_Has(info, "echo_level"));
+    CoSimIO_Info_SetInt(info, "echo_level", 6);
 
-    // CoSimIO_Info_SetInt(info, "echo_level", 6);
+    char buffer[BUFSIZ];
+    setbuf(stdout, buffer);
```
Comment (philbucher) [inline tests/co_sim_io/c/test_info_print.c:26]: and this one too :)

**Comment by pooyan-dadvand** [comment]
Let me check the Valgrind. Meanwhile, Tutorial 5 was not changed from CPP in this branch.

**Comment by philbucher** [comment]
@pooyan-dadvand all changes are in Master already, I also cleaned the tutorials in the meantime (check your whatsapp)

**Pull request merged**